SQLScripts/CreateTables_ATeam.sql

```diff
@@ -2,13 +2,26 @@ CREATE DATABASE Kainos_A;
 
 use Kainos_A;
 
-CREATE TABLE `roles` (
+CREATE TABLE `capability` (
+  `capabilityID` tinyint NOT NULL AUTO_INCREMENT,
+  `capability` varchar(40) DEFAULT NULL,
+  PRIMARY KEY (`capabilityID`)
+);
+
+CREATE TABLE `band` (
+  `bandID` tinyint NOT NULL AUTO_INCREMENT,
+  `bandName` varchar(40) NOT NULL,
```

Comment:
```suggestion
  `bandName` varchar(40) NOT NULL
```

Reply:
need a comma on the field before a primary key declaration